recipes/gtest/all/conanfile.py

```diff
@@ -18,6 +18,27 @@ class GTestConan(ConanFile):
     default_options = {"shared": False, "build_gmock": True, "fPIC": True, "no_main": False, "debug_postfix": 'd', "hide_symbols": False}
     _source_subfolder = "source_subfolder"
 
+    @property
+    def _minimum_cpp_standard(self):
+        if self.version == "1.8.1":
+            return 98
+        else:
+            return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        if self.version == "1.8.1":
+            return {
+                "Visual Studio": "13"
```

Comment:
Why 13 now? It was 12 before this modification.

Reply:
From officially supported list: https://github.com/google/googletest/tree/release-1.8.1#windows-requirements only 2015 and up are supported. 2013 is 12.

Reply:
Visual Studio 2015 = Visual Studio 14
Visual Studio 2013 = Visual Studio 12

Reply:
Hmmm both should be 14 then according to google as 1.8.1, 1.10.0 and head require 2015. Thanks, fixed.